Error while building docker image

Hi,

I am trying to build the image using dockerfile and getting below exception. Any help is appreciated

```
=> ERROR [builder 5/6] RUN mvn clean package -Passembly -Dmaven.test.skip --quiet                                                                                              575.8s
------                                                                                                                                                                                 
 > [builder 5/6] RUN mvn clean package -Passembly -Dmaven.test.skip --quiet:                                                                                                           
#8 575.6 [ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:3.15.3:build (default) on project debezium-server-bigquery-dist: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
#8 575.6 [ERROR]        [error]: Build step io.quarkus.deployment.index.ApplicationArchiveBuildStep#build threw an exception: org.jboss.jandex.UnsupportedVersion: Can't read Jandex index from /META-INF/jandex.idx: Can't read index version 13; this IndexReader only supports index versions 2-3,6-12
#8 575.6 [ERROR]        at io.quarkus.deployment.index.IndexingUtil$MetaInfJandexReader.apply(IndexingUtil.java:299)
#8 575.6 [ERROR]        at io.quarkus.deployment.index.IndexingUtil$MetaInfJandexReader.apply(IndexingUtil.java:275)
#8 575.6 [ERROR]        at io.quarkus.paths.PathTreeVisit.process(PathTreeVisit.java:39)
#8 575.6 [ERROR]        at io.quarkus.paths.OpenContainerPathTree.apply(OpenContainerPathTree.java:133)
#8 575.6 [ERROR]        at io.quarkus.paths.ArchivePathTree$OpenArchivePathTree.apply(ArchivePathTree.java:301)
#8 575.6 [ERROR]        at io.quarkus.paths.PathTreeWithManifest.apply(PathTreeWithManifest.java:75)
#8 575.6 [ERROR]        at io.quarkus.paths.SharedArchivePathTree$CallerOpenPathTree.apply(SharedArchivePathTree.java:162)
#8 575.6 [ERROR]        at io.quarkus.deployment.index.IndexingUtil.indexTree(IndexingUtil.java:74)
#8 575.6 [ERROR]        at io.quarkus.deployment.index.ApplicationArchiveBuildStep.lambda$addMarkerFilePaths$1(ApplicationArchiveBuildStep.java:251)
#8 575.6 [ERROR]        at io.quarkus.bootstrap.classloading.PathTreeClassPathElement.apply(PathTreeClassPathElement.java:136)
#8 575.6 [ERROR]        at io.quarkus.deployment.index.ApplicationArchiveBuildStep.addMarkerFilePaths(ApplicationArchiveBuildStep.java:238)
#8 575.6 [ERROR]        at io.quarkus.deployment.index.ApplicationArchiveBuildStep.scanForOtherIndexes(ApplicationArchiveBuildStep.java:144)
#8 575.6 [ERROR]        at io.quarkus.deployment.index.ApplicationArchiveBuildStep.build(ApplicationArchiveBuildStep.java:105)
#8 575.6 [ERROR]        at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
#8 575.6 [ERROR]        at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:856)
#8 575.6 [ERROR]        at io.quarkus.builder.BuildContext.run(BuildContext.java:256)
#8 575.6 [ERROR]        at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
#8 575.6 [ERROR]        at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
#8 575.6 [ERROR]        at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
#8 575.6 [ERROR]        at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
#8 575.6 [ERROR]        at java.base/java.lang.Thread.run(Thread.java:1583)
#8 575.6 [ERROR]        at org.jboss.threads.JBossThread.run(JBossThread.java:483)
```

--- a/debezium-server-bigquery-sinks/pom.xml
+++ b/debezium-server-bigquery-sinks/pom.xml
@@ -130,7 +130,7 @@
             <plugin>
                 <groupId>io.smallrye</groupId>
                 <artifactId>jandex-maven-plugin</artifactId>
-                <version>3.3.0</version>
+                <version>3.2.7</version>
                 <executions>
                     <execution>
                         <id>make-index</id>

